dirty_cat/_fuzzy_join.py

```diff
@@ -0,0 +1,176 @@
+"""
+Fuzzy joining tables using string columns.
+The principle is as follows:
+  1. We transform the key columns strings to vectors using CountVectorizer.
+  2. For each category, we use the nearest neighbor method to find its closest
+  neighbor and establish a match.
+  3. We match the tables using the previous information.
+Categories from the two tables that share many n-grams have greater
+probability of beeing matched together. The join is based on
+morphological similarities between strings.
+"""
+
+import pandas as pd
+import numpy as np
+from sklearn.feature_extraction.text import CountVectorizer, TfidfTransformer
+from sklearn.neighbors import NearestNeighbors
+
+
+def FuzzyJoin(left_table, right_table, on, return_distance=False, analyzer="char_wb",
```

Comment:
This is a function, it should be named without capitalization: "fuzzy_join" (it's a general convention: no function should have capitals in the name, and it applies to other places in the docs)